conference.js

```diff
@@ -719,6 +736,9 @@ export default {
      */
     init(options) {
         this.roomName = options.roomName;
+        const videoBlurEffectEnabled = APP.store.getState()['features/blur'].blurEnabled;
+
+        this._videoBlurScriptPromise = loadScript('libs/video-blur-effect.min.js');
```

Comment:
I thought the idea is to load the script only if it will be necessary. Is there a reason why you are always loading it?

Reply:
loadScript is now called only when flag is set or if effects.JitsiStreamBlurEffect is not defined